src/main/java/com/madgag/gif/fmsware/AnimatedGifEncoder.java

```diff
@@ -345,6 +372,30 @@ protected int findClosest(Color c) {
 	}
 	
 	/**
+	 * Returns index of palette color closest to c
+	 *
+	 */
+	protected int findExact (Color c) {
+		if (colorTab == null) return -1;
+		int r = c.getRed() & 0xff;
+		int g = c.getGreen() & 0xff;
+		int b = c.getBlue() & 0xff;

```

Comment:
You're right. My mistake... I didn't check get[Red,Green,Blue] documentation where it clearly states that the range is 0..255, so no need to bitmask with 0xff. Fixed.